okapi-common/src/test/java/org/folio/okapi/common/refreshtoken/tokencache/MapExpiryTest.java

```diff
@@ -43,10 +42,13 @@ public void testExpiryPut() {
   @Test
   public void testCapacityWithExpiration() {
     ExpiryMap tk = new ExpiryMapImpl(2);
-    tk.put("user1", "v1", System.currentTimeMillis() + 5);
-    tk.put("user2", "v2", System.currentTimeMillis() + 5);
-    Awaitility.await().pollInterval(1, TimeUnit.MILLISECONDS).atMost(10, TimeUnit.MILLISECONDS)
-        .until(() -> tk.get("user2") == null);
+    var expire = System.currentTimeMillis() + 5;
+    tk.put("user1", "v1", expire);
+    tk.put("user2", "v2", expire);
+    while (System.currentTimeMillis() <= expire) {
+      // wait
+    }
+    assertThat(tk.get("user2"), is(nullValue()));
```

Comment:
Why is the approach with awaitlity brittle? Is it that the time unit of 1 millisecond is too small to work reliably? If so why not just increase the time unit to something a bit more reasonable and keep awaitility instead of the while loop?